core/error_codes.rs

```diff
@@ -1,6 +1,6 @@
 use anyhow::Error;
 
-pub(crate) fn get_error_code(err: &Error) -> Option<&'static str> {
+pub fn get_error_code(err: &Error) -> Option<&'static str> {
```

Comment:
This is also introducing circular dependency between `deno_core/` and `ops/`

Reply:
Ditto